modules/sandstorm/manifests/init.pp

```diff
@@ -1,5 +0,0 @@
-class sandstorm {
```

Comment:
I guess sandstorm's manifest can just be removed for now and puppet runs on it will fail until it is completely shut down? I think it would be better to wait to disable sandstorm at all until the new system is fully working.